sources/web/datalab/common.d.ts

```diff
@@ -87,6 +87,26 @@ declare module common {
      * If true, allow HTTP requests via websockets.
      */
     allowHttpOverWebsocket: boolean;
+    
+    /**
+     * Whether to automatically back up user's contents dir to GCS
+     */
+    enableAutoGCSBackups: boolean;
```

Comment:
Makes sense.

However I think these should still be part of the global settings, with the userSettings optionally overriding them. This way we can keep our defaults in one place, and the user starts out with an empty settings file that they modify however they need. We then modify the loadUserSettings function to first try loading from the user's config dir, then default to the global settings file. Thoughts?